frontend/routes/components/filterPanel/filterPanel.component.tsx

```diff
@@ -63,9 +64,10 @@ interface ISelectedFilter {
 }
 
 interface IProps {
-	filters: IFilter[];
+	filters?: IFilter[];
 	onChange: (selectedFilters) => void;
 	selectedFilters: any[];
+	hideFiltersMenu?: boolean;
```

Comment:
You no need to add `Filters` into the name. It is filtersPanel, so it is logical that this prop is related to it. :P